router/router.go

```diff
@@ -200,10 +203,14 @@ func New(cfg *config.Configuration, rateConvertor *currency.RateConverter) (r *R
 	// Metrics engine
 	r.MetricsEngine = metricsConf.NewMetricsEngine(cfg, openrtb_ext.CoreBidderNames(), syncerKeys, moduleStageNames)
 	shutdown, fetcher, ampFetcher, accounts, categoriesFetcher, videoFetcher, storedRespFetcher := storedRequestsConf.NewStoredRequests(cfg, r.MetricsEngine, generalHttpClient, r.Router)
-	// todo(zachbadgett): better shutdown
-	r.Shutdown = shutdown
 
-	analyticsRunner := analyticsBuild.New(&cfg.Analytics)
+	r.AnalyticsRunner = analyticsBuild.New(&cfg.Analytics)
+
+	// todo(zachbadgett): better shutdown
+	r.Shutdown = func() {
+		shutdown()
+		r.AnalyticsRunner.Shutdown()
```

Comment:
going to add unit test for this shutdown function so that when router shutdown is invoked, the analytic runner's shutdown is invoked as well